atlassian_jwt_auth/signer.py

```diff
@@ -9,11 +9,15 @@
 
 class JWTAuthSigner(object):
 
-    def __init__(self, issuer, private_key_retriever, **kwargs):
+    def __init__(self, issuer, private_key_retriever,
```

Comment:
I don't understand the benefit of using kwargs unpacking? As far as I can tell it just makes things less explicit and makes it harder to see at a glance what's going on.